### PR TITLE
Fits the user story, no RSpec for app but about 98% working

### PR DESCRIPTION
**Pull request by Vanboylan**

### CODE REVIEW

File: lib/app.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+def interactive_menu
+    loop do
+        puts "What would you like to do?"
+        puts ("1. List all items")
+        puts ("2. create new item")
+        puts ("3. List all orders")
+        puts ("4. create new order")
+        puts ("9. Exit")
+        answer = gets.chomp
+        case answer
```
Comment (GuillerminaLorenzo) [inline lib/app.rb]: Creating another method could be useful to avoid having to many lines in one method

File: lib/item_repository.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      new_item.name = item['name']
+      new_item.unit_price = item['unit_price']
+      new_item.quantity = item['quantity']
+      items << new_item
+    end
+    return items
+  end
+
+  def create
+    @io.puts "What is the item name?"
```
Comment (GuillerminaLorenzo) [inline lib/item_repository.rb]: You could have this @io.puts in the app.rb file

File: lib/order_repository.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        new_order.order_date = order['order_date']
+        new_order.customer_name = order['customer_name']
+        new_order.item_id = order['item_id']
+        orders << new_order
+    end
+    return orders
+  end
+
+  def create
+    @io.puts "What is the customer name?"
```
Comment (GuillerminaLorenzo) [inline lib/order_repository.rb]: this method might be long and confusing. Maybe you could include some parts of this method in the app.rb

File: spec/item_repository_spec.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        reset_items_table
+      end
+  it "constructs" do
+    io = double(:io)
+    repo = ItemRepository.new(io)
+    result = repo.all
+    expect(result.length).to eq 5
+  end
+
+  it "creates a new item" do
```
Comment (GuillerminaLorenzo) [inline spec/item_repository_spec.rb]: good job testing @io

File: spec/seeds_items.sql [hunk 5]
```diff
@@ -0,0 +1,41 @@
+TRUNCATE TABLE items, orders, items_orders RESTART IDENTITY;
+
+INSERT INTO items (id, name, unit_price, quantity) VALUES(1, 'shark vacuum', 50, 100);
```
Comment (GuillerminaLorenzo) [inline spec/seeds_items.sql:3]: avoid repetition

File: lib/app.rb (hunk 1 above)
Comment (GuillerminaLorenzo) [inline lib/app.rb]: having more than one method here could be useful to avoid having too many lines in one method

File: lib/order_repository.rb [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    now = time.strftime("%Y-%m-%d")
+    params = [name, now, item_id.to_i]
+    DatabaseConnection.exec_params(query, params)
+    query = 'SELECT quantity FROM items WHERE id = $4;'
+    params = [item_id.to_i]
+    quantity = DatabaseConnection.exec_params(query, params)
+    query = 'UPDATE items SET quantity = $5 WHERE id = $6;'
+    params = [quantity.to_i, item_id.to_i]
+    DatabaseConnection.exec_params(query, params)
+    @io.puts "Order created and quantity updated"
```
Comment (iniffur) [inline lib/order_repository.rb]: Repetition - could potentially be refactored

File: lib/order_repository.rb [hunk 7]
```diff
@@ -0,0 +1,41 @@
+require '../lib/order'
+
+class OrderRepository
+  def initialize(io)
+    @io = io
+  end
+
+  def all
+    query = 'SELECT * FROM orders;'
```
Comment (GuillerminaLorenzo) [inline lib/order_repository.rb:6]: great method

File: lib/app.rb [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+        when '1'
+            items.all
```
Comment (GuillerminaLorenzo) [inline lib/app.rb]: think about what you are returning in the method all and create and how should the output look like

File: lib/app.rb [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+        when '2'
+            items.create
+        when '3'
+            orders.all
+        when '4'
+            orders.create
+        when '9'
+            exit
+        else
+            "Invalid selection"
```
Comment (GuillerminaLorenzo) [inline lib/app.rb]: good but you'll need something to return it

File: spec/order_repository_spec.rb [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+        reset_orders_table
+      end
+  it "constructs" do
+    io = double(:io)
+    repo = OrderRepository.new(io)
+    result = repo.all
+    expect(result.length).to eq 11
+  end
+
+  it "creates a new order" do
```
Comment (GuillerminaLorenzo) [inline spec/order_repository_spec.rb]: good test

**Review by GuillerminaLorenzo — COMMENTED** [review]
nice!

**Pull request closed without merging**